src/error.rs

```diff
@@ -94,6 +97,12 @@ impl ValidationError {
             "{}:{type_name}:{what}:{in_what_type}",
             location_sort_key(location)
         );
+        if location.is_none() {
+            eprintln!(
+                "WARN: An error is missing a span and will be printed without context, file name, and line number. \
+                This is a bug. Please report it with a piece of Rust code that triggers it at: {NEW_ISSUE_URL}"
```

Comment:
```suggestion
                This is a bug. Please report it with a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example) that triggers it at: {NEW_ISSUE_URL}"
```

Reply:
It's very difficult to come up with one of these without sprinkling `cargo-check-external-types` with carefully placed panics that output the type ID that caused the issue, so I don't really want to require that. It's easy for me to do, but hard for someone not familiar with the tool's code to do.